cmd/operator/main.go

```diff
@@ -39,7 +39,10 @@ import (
 	//+kubebuilder:scaffold:imports
 )
 
-const otelServiceName = "kubearchive.operator"
+const (
+	otelServiceName   = "kubearchive.operator"
+	enablePprofEnvVar = "KUBEARCHIVE_ENABLE_PPROF"
```

Comment:
I think you can remove this const since we're now using `observability.EnablePprofEnvVar`

Reply:
Ops, my bad. Removed.